http-proxy/Cargo.toml

```diff
@@ -1,21 +1,14 @@
 [package]
 name = "http-proxy"
 version = "0.1.0"
-authors = ["Nikolay Kim <fafhrd91@gmail.com>"]
-edition = "2018"
+authors = ["Rotem Yaari"]
```

Comment:
is it needed to change the author? In any case why not listing both? (real question, don't know how the community handle crate authors)

Reply:
I've authored several examples and haven't bothered adding myself to the list.  Nikolay has maintained the repo after initial submissions are made.   An original author plus Nikolay seems fair.

Thoughts?

Reply:
Sure, I will add myself with Nikolai going first of course. The reason for this is that I just moved the full http-proxy example that I wrote from scratch over to this new one...